Gemfile

```diff
@@ -6,14 +6,17 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 ENV['BOLT_DISABLE_ANALYTICS'] = 'true'
 
 gemspec
-gem "hocon"
-gem "json-schema"
-gem "puma"
-gem "rack"
-gem "rails-auth"
-gem "sinatra"
+
+# Bolt server gems are managed here not in the gemspec
+gem "hocon", '>= 1.2.5'
```

Comment:
was considering putting this in a gem group to make that clearer.

Reply:
I've never found gem groups to be that useful. Unless we're super concerned about bloat in some test matrix cells I wouldn't bother.